### PR TITLE
Adding mobile sso

### PR DESCRIPTION
**Pull request by gddhatdisqus**

### CODE REVIEW

File: mobile-sso/js-server-demo/index.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    email: 'person327@example.com',
+    avatar:'https://i.imgur.com/AYgoB04.jpg',
+    url: 'example7.com',
+    profile_url:'https://example.com/providedProfileUrl-1234532329'
+  },
+  // test user 4
+  {
+    id:'1234523232329',
+    username: 'uniqueperson122327',
+    email: 'dmatt@disqus.com',
```
Comment (hlgott1593) [inline mobile-sso/js-server-demo/index.js:81]: Can you remove all the dmatt and taylan example emails and references please.
Does there need to be 16 test users?

Comment (gddhatdisqus) [reply mobile-sso/js-server-demo/index.js]: done

**Review by hlgott1593 — APPROVED** [review]

File: mobile-sso/swift-demo/test/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        let url = components.url!
+        mywebview.load(URLRequest(url: url))
+    }
+    
+    func login() {
+        let parameters = [
+            "email": "dmatt+9898@disqus.com",
+            "password": "pwd"
+        ]
+//        let parameters = [
```
Comment (hlgott1593) [inline mobile-sso/swift-demo/test/ViewController.swift:41]: this seems like it needs to be cleaned up

File: mobile-sso/swift-demo/test/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        }
+
+        if (url.absoluteString.contains("/login-success") ||
+            (url.absoluteString.contains("/logout"))) {
+            decisionHandler(.cancel)
+            var components = URLComponents()
+            components.scheme = "http"
+            components.host = "localhost"
+            components.port = 5000
+//            components.scheme = "https"
```
Comment (hlgott1593) [inline mobile-sso/swift-demo/test/ViewController.swift:99]: also here. or is this intended?

Comment (gddhatdisqus) [reply mobile-sso/swift-demo/test/ViewController.swift:99]: intended

File: mobile-sso/swift-demo/test/ViewController.swift (hunk 2 above)
Comment (gddhatdisqus) [reply mobile-sso/swift-demo/test/ViewController.swift:41]: intended

**Pull request merged**